app/services/podcasts/get_episode.rb

```diff
@@ -36,5 +38,13 @@ def update_published_at(episode, item_data)
     rescue ArgumentError, NoMethodError => e
       Rails.logger.error("not a valid date: #{e}")
     end
+
+    def cache_episode_data(episode_cache_key, item_data)
+      Rails.cache.write(episode_cache_key, item_data.to_h)
```

Comment:
I think this only need to have `item_data` as an argument, since it can call `cache_key` by itself, which would simplify the call-site.

In case we want to keep the flexibility of being able to override the cache key (though it seems we only have a single caller), we can make it a keyword argument with a default value, otherwise I'd just inline it:

```ruby
# Ruby keyword arguments can use other arguments even in method calls
def cache_episode_data(item_data, episode_cache_key: cache_key(item_data))
  Rails.cache.write(episode_cache_key, item_data.to_h)
end

# or
def cache_episode_data(item_data)
  episode_cache_key = Digest::SHA1.hexdigest(item_data.to_s)
  Rails.cache.write(episode_cache_key, item_data.to_h)
end
```

Reply:
The problem with this is that we need the cache key to send to the worker above in line 17 so we need access to it in the `call` method

Reply:
Ah, I missed that. Good morning, Michael. I guess in that case I'd just inline the `Rails.cache.write`, the method doesn't seem to offer any benefit over that.

```ruby
        episode_cache_key = cache_key(item_data)
        Rails.cache.write(episode_cache_key, item_data.to_h)
        PodcastEpisodes::CreateWorker.perform_async(podcast.id, episode_cache_key)
```